projects/plugins/jetpack/extensions/blocks/wpcom-login/block.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "jetpack/wpcom-login",
+	"category": "theme",
+	"title": "WordPress.com Login/out",
+	"description": "Show WordPress.com login & logout links.",
+	"keywords": [ "login", "logout", "form", "WordPress.com" ],
+	"textdomain": "jetpack",
+	"icon": "<SVG xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><Path d='M11 14.5l1.1 1.1 3-3 .5-.5-.6-.6-3-3-1 1 1.7 1.7H5v1.5h7.7L11 14.5zM16.8 5h-7c-1.1 0-2 .9-2 2v1.5h1.5V7c0-.3.2-.5.5-.5h7c.3 0 .5.2.5.5v10c0 .3-.2.5-.5.5h-7c-.3 0-.5-.2-.5-.5v-1.5H7.8V17c0 1.1.9 2 2 2h7c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2z' /></SVG>",
```

Comment:
I don't believe `SVG`, `viewBox`, and `Path` will work, it's going to have to be valid svg I think.